src/redis_bitmap.cc

```diff
@@ -146,16 +146,27 @@ rocksdb::Status Bitmap::GetString(const Slice &user_key, const uint32_t max_btos
 
     //  If you setbit bit 0 1, the value is stored as 0x01 in Kvrocks but 0x80 in Redis.
     // So we need to swap bits is to keep the same return value as Redis.
+    static const uint8_t swap_table[256] = {
```

Comment:
Would you like to add some comments to describe how to generate the table? i.e. the correspending mapping definition:
```
swap_table(x) = (x & 0x80 >> 7) | ...
```

Reply:
Yeah, you can actually use templates. 

For more details, you can refer to [this commit in doris](https://github.com/apache/doris/commit/f49284036e1ed2bb5597cd6472720552a9c19054), which has samiliar ideas.

The commit above is written in c++17, so some modification is needed.

Reply:
I think `swap_table` is NOT easy to maintain and previous way looks better than now. You change this for performance concern?

Reply:
@ColinChamber Thanks for your improvement report, it also makes sense for me since it can improve the performance a lot.

Reply:
only 80.39 requests per second?